### ISSUE TITLE
how to train on my own database?

### ISSUE THREAD

**Issue by manvirvirk** [4uiiurz1/pytorch-nested-unet#10]
can anyone help??

**Comment by 404leng** [comment]
do you train dsb2018 success?

**Comment by manvirvirk** [comment]
no, i want to train on retina database. i have small database containing retina images.

**Comment by 4uiiurz1** [comment]
@manvirvirk @404leng 
I add support for multi-class segmentation. Please pull the latest commit and try again. Let me know if you have any problems.

**Comment by baaaohaaan** [comment]
It may be dumb, but must my own dataset contains .jpg or .png files only? how about .nii or .nii.zip files? cuz I tried to train dataset with .nii.zip files, but it failed. Hope someone could help me. Appreciated!
